app/assets/social.json

```diff
@@ -10,23 +10,23 @@
 	"email_notifications": {
 		"contest_starting": {
 			"subject": "Kontes dimulai dalam waktu #{time_text}",
-			"text": "Hanya mengingatkan saja, #{@contest} akan dimulai #{time_text} kemudian. Siapkan segala peralatan dan alat perang Anda!"
+			"text": "Hanya mengingatkan saja, #{@contest} akan dimulai #{time_text} kemudian. Siapkan segala peralatan dan alat perang Anda!\n\nKlik link ini untuk tidak menerima lagi email dari kami\n#{link}"
```

Comment:
I think something like this would be better:

> Anda menerima email ini karena Anda subscribe ke notifikasi kontes kami. Klik link ini untuk berhenti menerima email dari kami: #{link}

Basically it's nice to tell them why they receive the email.

Also, can you put this message below the salutation? There's a salutation that is automatically attached to our emails ("Salam,\nTim KTO Matematika").

Reply:
Got it

Reply:
I'm sorry, but i guess if you put it below salutation, not only email, but certificate, pdf, problems etc will show unsubscribe link which is bad. CMIIW

Reply:
Yeah so the point is to figure out how to only show the unsubscribe link for these emails :)

Reply:
I think the one and only way is before, when i put link in the text, not the template

Reply:
What do you mean by "text" and "template"?

Reply:
before, i put link in text like contest_started.text, contest_ended.text, and 3 more event, i forget. But you want to me put the link under salam tim kto matematika which is located in email_templates. so i guess the right way is put in eact event text

Reply:
each

Reply:
not under salam kto matematika that located in email_template

Reply:
So the current system won't be able to accommodate adding additional text below the salutation according to the email template. You'll need to change the system so that you can pass in additional text below the salutation.